autonomy/configurations/base.py

```diff
@@ -391,4 +385,5 @@ def try_to_process_nested_fields(  # pylint: disable=too-many-locals
 PACKAGE_TYPE_TO_CONFIG_CLASS = {
     **_PACKAGE_TYPE_TO_CONFIG_CLASS,
     PackageType.SERVICE: Service,
+    PackageType.CONNECTION: ConnectionConfig,
```

Comment:
I think this can go. `PackageType.CONNECTION` is already specified in `_PACKAGE_TYPE_TO_CONFIG_CLASS`

Reply:
https://github.com/valory-xyz/open-aea/pull/298/files#diff-9d90463edcd0573846e5906abe4e7def07e18ad71ed8b7ae1da9f8e33330ec4bR33-R39